libs/coin-modules/coin-sui/src/network/sdk.ts

```diff
@@ -41,6 +41,8 @@ const fetcher = (url: Inputs[0], options: Inputs[1], retry = 3): Promise<Respons
       ...options.headers,
       "X-Ledger-Client-Version": getEnv("LEDGER_CLIENT_VERSION"),
     };
+
+    options.signal = AbortSignal.timeout(60000);
```

Comment:
Are we sure AbortSignal is available on all platform especially for hermes on RN ?

Reply:
hey, closing this actually because of that, in theory AbortSignal.timeout is available since node 18, but it's failing, I am closing this as it seems no more issues with timeout ATM